src/main/java/seedu/momentum/storage/JsonAdaptedProject.java

```diff
@@ -63,10 +68,24 @@ public Project toModelType() throws IllegalValueException {
         if (!Name.isValidName(name)) {
             throw new IllegalValueException(Name.MESSAGE_CONSTRAINTS);
         }
+        if (description == null) {
+            throw new IllegalValueException(String.format(
+                    MISSING_FIELD_MESSAGE_FORMAT, Description.class.getSimpleName()));
+        }
+        if (!Description.isValidDescription(description)) {
+            throw new IllegalValueException(Description.MESSAGE_CONSTRAINTS);
+        }
         final Name modelName = new Name(name);
 
+        if (description == null) {
+            throw new IllegalValueException(
+                    String.format(MISSING_FIELD_MESSAGE_FORMAT, Description.class.getSimpleName()));
+        }
+        final Description modelDescription = new Description(description);
+
+
```

Comment:
I think the code here should be rearranged so that it is a little clearer. The `modelName` variable should be delared after the `if (!Name.isValidName(name))` block